static/style.css

```
@@ -2249,3 +2249,77 @@ details[open] > .context-card-head .collapsed-notes {
+    background: black;
+}
+
+/* PDF placeholder: browser-native PDF embedding is inconsistent across
+   vendors, so we show a compact label + filename instead. */
+.context-preview-pdf-icon {
+    padding: var(--spacing-md);
+    background: var(--bg-secondary);
+    border-radius: var(--radius);
+    margin-bottom: var(--spacing-sm);
+    font-size: 0.9rem;
+    word-break: break-word;
```

Stylelint (17.10.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)